posthog/api/test/test_decide.py

```diff
@@ -426,6 +427,26 @@ def test_session_recording_network_payload_capture_config(self, *args):
         response = self._post_decide().json()
         self.assertEqual(response["sessionRecording"]["networkPayloadCapture"], {"recordHeaders": True})
 
+    def test_session_recording_masking_config(self, *args):
+        self._update_team(
+            {
+                "session_recording_opt_in": True,
+            }
+        )
+
+        response = self._post_decide().json()
+        self.assertEqual(response["sessionRecording"]["masking"], None)
```

Comment:
nit picking that the `assert blah is None` or assert foo == bar` syntax gives better output when failing